src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Options/RequestTimeoutOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Options;
+
+public class RequestTimeoutOptions
+{
+    public const string Section = "RequestTimeouts";
```

Comment:
This should be `RequestTimeout` to align with the class

Reply:
done :)